clippy_lints/src/utils/conf.rs

```diff
@@ -238,6 +241,15 @@ pub fn read(path: &Path) -> (Conf, Vec<Error>) {
                 errors.push(Error::Toml(cyc_err));
             }
 
+            let msrv: Option<&String> = toml_ref.msrv.as_ref();
+
+            let _msrv = msrv.and_then(|s| {
+                parse_msrv(s, None, None).or_else(|| {
+                    errors.push(Error::Toml(format!("`{}` is not a valid Rust version", msrv.unwrap())));
+                    None
+                })
+            });
```

Comment:
Oh, this is only there for error checking and nothing else. Is this still necessary here, if we parse the MSRV in `lib.rs`? We have access to a `Session` in `lib.rs`, so we could emit an error there and don't need to do this here.

Reply:
Yeah we can do this in `lib.rs` too as we have access to a `Session`. I just saw that usages of the deprecated `cyclomatic_complexity_threshold` are checked here so thought this is the right place for such validations

Reply:
Yeah, I initially thought so, too. But now I think we should just do this in `lib.rs`.